cpp/Platform.Data.Doublets/ILinksExtensions.h

```diff
@@ -414,9 +414,9 @@ namespace Platform::Data::Doublets
             return usagesAsSource + usagesAsTarget;
         }
 
-    //    template<typename TStorage>
-    //    static bool HasUsages(TStorage& storage, typename TStorage::LinkAddressType linkAddress) { return Comparer<typename TStorage::LinkAddressType>.Default.Compare(Count(storage)Usages(linkAddress), 0) > 0; }
-    //
+        template<typename TStorage>
+        static bool HasUsages(TStorage& storage, typename TStorage::LinkAddressType linkAddress) { return Comparer<typename TStorage::LinkAddressType>.Default.Compare(Count(storage)Usages(linkAddress), 0) > 0; }
```

Comment:
Comparer class exists in C#, not in C++. As I said earlier: C++ can easily compare TLinkAddresses. Just return true if CountUsages returns > 0